bin/mutatex

```diff
@@ -299,25 +298,63 @@ try:
 except:
     exit(1)
 
+#get the original PDB list
+pdbs_list = [ os.path.join(main_dir, pdb) for pdb in pdbs_list ]
+original_residues_dictionary = {}
+
+for pdb in pdbs_list:
+    this_dirname = working_directory+"/"+os.path.splitext(os.path.basename(pdb))[0]
+    try:
+        safe_makedirs(this_dirname)
+    except:
+        log.error("PDB %s will be ignored." % pdb)
+        pdbs_list.remove(pdb)
+        continue
+    try:
+        key = pdb.split("/")[-1].split("_checked")[0]
```

Comment:
The challenge here: 

example with P02655.pdb

For file 1 - the non-repaired file, name: '`P02655_model0_checked.pdb`'
old_key = `pdb.split('/')[-1].split('_checked')[0]`
new_key = `os.path.splitext(os.path.basename(pdb))[0]`

results:
old key: `P02655_model0`
new key: `P02655_model0_checked`

For file 2 - the repaired file, name: '`P02655_model0_checked_Repair.pdb`'
old_key = `pdb.split('/')[-1].split('_checked')[0]`
new_key = `os.path.splitext(os.path.basename(pdb))[0]`

new key: `P02655_model0_checked_Repair`
old key: `P02655_model0`

keys are used in the dictionary to find comparison: 
`P02655_model0_checked != P02655_model0_checked_Repair`

Since os.path.splitext(os.path.basename) splits to remove the file extention, and does not seem to be very flexible when the basenames are different. e.g. we need that the `first`_model`number`_ ... can be anything in principle.

Reply:
we can also: `os.path.splitext(pdb)[0].split("_checked")[0]` if you prefer.

Reply:
that was also my first attempt, but in principle the `splitext` isn't useful since we are splitting the extension out anyway.

actually I have another concern - what if the original filename contains `_checked`? Do we risk splitting out part of the filename?

e.g.

```my_pdb_checked -> my_pdb_checked_model0_checked -> my_pdb
     my_pdb_checked -> my_pdb_checked_model1_checked -> my_pdb```

what if we change perspective, and keep `os.path.splitext(os.path.basename(pdb))[0]` as key, and remove the trailing `_Repair` from the repaired PDB? Since we know that is always going to be there